react_app/src/App.scss

```diff
@@ -0,0 +1,183 @@
+.App {
+  font-family: sans-serif;
+  text-align: center;
+}
+
+html {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
```

Comment:
hmmm not sure about this alignment. Don't have a strong feeling about it, but is this in the style guide? Also, I think since ya'll are demoing via *Chrome*books, we don't need to worry about legacy browser support. :P

Reply:
Fixed! just removed webkit stuff